src/coreclr/vm/methodtable.cpp

```diff
@@ -4963,7 +4963,10 @@ OBJECTREF MethodTable::FastBox(void** data)
     if (IsNullable())
         return Nullable::Box(*data, this);
 
-    OBJECTREF ref = Allocate();
+    // Allocate without triggering cctors - boxing should not trigger cctors
```

Comment:
```suggestion
    // MethodTable::Allocate() triggers cctors, so to avoid that we
    // allocate directly without triggering cctors - boxing should not trigger cctors.
```